lib/utils/dataflow/src/index.js

```diff
@@ -280,12 +281,18 @@ const postError = (oid, res) => {
 // Build a list of output docs
 const buildOutputs = (itype, idoc, otype, odocs, okeys, otimes, now) => {
```

Comment:
The reason is because I think the id should be based on the input key rather than the output key.
Instead of setting id = dbclient.kturi(okeys[i], otimes[i]), I set the id of the error doc to be based on the input doc's id, hence id = [idoc.id, i].join('/') where i = the doc index of the input array.

I think the reason why I didn't attach processed_id is because I thought of it as a field to detect duplicate, and I thought that the error doc is an unprocessed doc because it is just the input doc + error and reasons field.

Should we just leave them to be the same?